scripts/playbook.yml

```diff
@@ -29,11 +31,15 @@
         - cmake
         - python-psycopg2
         - libselinux-python
-        - git
         - bash-completion
         - vim
-    - name: install bower package manager
-      npm: name=bower global=yes
+    - name: install npm packages needed
+      npm: name={{ item }} global=yes
+      with_items:
+        - bower
+        - yarn
+        - gulp-cli
+        - webpack
```

Comment:
Maybe it's needed for "developer setup" then?? I'm not familiar enough with the differences between our appliances vs developer setup, so I'll leave that up to someone who knows...